.github/workflows/core-e2e-test.yml

```diff
@@ -0,0 +1,33 @@
+name: Core e2e tests
```

Comment:
i guess maybe we don't need this action? Seems like all the end-to-end test stuff is outdated since we have it in the frontend now